scripts/lint.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+
+set -eu
+
+golint --set_exit_status
```

Comment:
```suggestion
golint -set_exit_status
```

Either way works though.

Reply:
fwiw I tend to use the `--` form for flags that are longer than 1 char because it's more idiomatic wrt command lines, but afaik the single `-` is more idiomatic wrt Go so *shrug*
(not making a decision here, just wanted to weight in with my opinion on the matter)

Reply:
I totally agree. But `-set_exit_status` is just what's from the help.